composer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "ezsystems/content-on-the-fly",
```

Comment:
Ok, was thinking we should be somewhat consistent with fulltext search prototype bundle.